### DISCUSSION TITLE
Black screen triggering motion

### DISCUSSION THREAD

**Discussion by MildarAA** [blakeblackshear/frigate#2926] [General]
Hi. I have a problem that when it is night and there is no light in room camera is recording frigate start to wildly detect screen as motion. I whink it is because video noise becomes too significant. Any idea how to change settings not to trigger darkness as motion?

thx

**Comment by NickM-27** [comment]
Just curious, what symptoms are you seeing that indicates lots of motion? Does your 24/7 recording include a lot of segments with black screen and nothing else?

**Reply by MildarAA** [reply]
> Just curious, what symptoms are you seeing that indicates lots of motion? Does your 24/7 recording include a lot of segments with black screen and nothing else?

I watched camera debug screen with motion boxes enabled. With light on everything acted as expected. With light off motion boxes are all over the place. Also CPU usage (no coral yet) is 4x bigger indicating motion is triggering AI procesing.

**Reply by MildarAA** [reply]
I am thinking if some type of filter doing quick pixel color average every second that would disable motion(detect?) would be worth it because at the moment I am unable to find the right motion setting.

**Reply by blakeblackshear** [reply]
It's probably the dynamic contrast amplifying the small differences in the almost black image. It would make even a tiny change dramatic.

**Reply by MildarAA** [reply]
You were right. I commented out dynamic contrast in motion.py for now (Thank god it is in easy to change python!) as it is not needed in my specific settings where there is motion induced light fully illuminating area. Consider adding to config either some type of black screen treshold or full dynamic contrast turn off option.

Anyway thanks for turning me in right direction.

**Reply by hawkeye217** [reply]
I too see the same thing in a couple of dark rooms where I'm using cheap cams with Frigate for presence detection. I opened [this issue](https://github.com/blakeblackshear/frigate/issues/2835) to be able to enable/disable motion detection (much like detection, recordings, and snapshots can be enabled/disabled), which I could just automate via Home Assistant. Blake also [indicated](https://github.com/blakeblackshear/frigate/issues/2863#issuecomment-1059887054) he may make adjustments to the newer dynamic contrast code introduced in 0.10 as well. 

I've commented out the code in motion.py in the meantime as well.
